.github/workflows/s390x-cxx20.yml

```diff
@@ -0,0 +1,29 @@
+name: Ubuntu s390x (GCC 12, CXX 20)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: uraimo/run-on-arch-action@v3
+        name: Test
+        id: runcmd
+        with:
+          arch: s390x
+          githubToken: ${{ github.token }}
+          distro: ubuntu22.04
+          install: |
+            apt-get update -q -y
+            apt-get install -y cmake make g++-12
+          run: |
+            CXX=g++-12 cmake -DSIMDUTF_CXX_STANDARD=20 -DCMAKE_BUILD_TYPE=Release -B build
+            cmake --build build -j=2
```

Comment:
maybe use `$(nproc)` instead?

Reply:
@pauldreik 

> I will wait for the others to chime in.

You are in charge Paul!!!

I'm busy on the patch work.

Reply:
ok, @selfisekai please switch the existing s390x job to C++20 instead of adding a new one!